framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -3586,35 +3586,33 @@ public AnnotatedTypeMirror widenToUpperBound(
     }
 
     /**
-     * Returns the functional interface and the function type that this member reference targets.
+     * Returns the function type that this member reference targets.
      *
      * <p>The function type is the type of the single method declared in the functional interface
      * adapted as if it were invoked using the functional interface as the receiver expression.
      *
      * <p>The target type of a member reference is the type to which it is assigned or casted.
      *
      * @param tree member reference tree
-     * @return the functional interface and the function type that this method reference targets
+     * @return the function type that this method reference targets
      */
-    public Pair<AnnotatedDeclaredType, AnnotatedExecutableType> getFnInterfaceFromTree(
-            MemberReferenceTree tree) {
-        return getFnInterfaceFromTree((Tree) tree);
+    public AnnotatedExecutableType getFunctionTypeTree(MemberReferenceTree tree) {
+        return getFnInterfaceFromTree(tree).second;
     }
 
     /**
-     * Returns the functional interface and the function type that this lambda targets.
+     * Returns the function type that this lambda targets.
      *
      * <p>The function type is the type of the single method declared in the functional interface
      * adapted as if it were invoked using the functional interface as the receiver expression.
      *
      * <p>The target type of a lambda is the type to which it is assigned or casted.
      *
      * @param tree lambda expression tree
-     * @return the functional interface and the function type that this lambda targets
+     * @return the function type that this lambda targets
      */
-    public Pair<AnnotatedDeclaredType, AnnotatedExecutableType> getFnInterfaceFromTree(
-            LambdaExpressionTree tree) {
-        return getFnInterfaceFromTree((Tree) tree);
+    public AnnotatedExecutableType getFunctionTypeTree(LambdaExpressionTree tree) {
```

Comment:
The method name is a bit odd - it sounds like it would return a type. Other methods below use `FromTree` to make this clearer.
So how about `getFunctionTypeFromTree` for this one? Or `getFunctionTypeFromLambdaTree`?

Reply:
Similar comment for `getFunctionTypeTree` above.